src/Display/Display.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+const DisplayStyles = styled.div`
+  font-weight: 500;
+  font-size: 48px;
+`;
+
+export default function Display({ children, style }) {
+  return <DisplayStyles style={style}>{children}</DisplayStyles>;
+}
+
+Display.propTypes = {
+  children: PropTypes.string.isRequired,
```

Comment:
The HTML standard says the childen of a header tag can be any phrasing content: https://html.spec.whatwg.org/multipage/dom.html#phrasing-content

So I think the children proptypes for all of these should be something like:
```
PropTypes.oneOfType([
    PropTypes.arrayOf(PropTypes.node),
    PropTypes.node
  ]),
```

So it children can be other nodes.